cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,62 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
-
-  });
+const faker = require("faker");
 
+describe('Settings page', () => {
   beforeEach(() => {
+    cy.task('db:clear');
+    cy.registerNewUser().then(user => {
+      cy.login(user).then(() => user);
+    }).as('user');
 
+    cy.visit('#/settings')
   });
 
   it('should provide an ability to update username', () => {
-
+    const newUsername = faker.internet.userName();
+    cy.get('@user').then(user => {
+      cy.getByDataCy('username-settings')
+        .clear().type(newUsername);
+        cy.getByDataCy('update-settings').click();
+      cy.contains('.swal-modal', 'Update successful!').should('exist');
```

Comment:
You have repeated the same lines of code in multiple test cases. Instead you can move these to a new function to make your code DRY (Don't Repeat Yourself).